registration-processor/core-processor/registration-processor-finalization-stage/src/main/java/io/mosip/registration/processor/stages/finalization/stage/FinalizationStage.java

```diff
@@ -58,6 +59,7 @@ public class FinalizationStage extends MosipVerticleAPIManager{
 	/** stage properties prefix */
 	private static final String STAGE_PROPERTY_PREFIX = "mosip.regproc.finalization.";
 	private static final String USER = "MOSIP_SYSTEM";
+	private static final String ID_REPO_KEY_MANAGER_ERROR = "IDR-IDS-003";

```

Comment:
Please remove here as it is already present in draft service

Reply:
done